curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-string-and-array-methods-by-building-a-music-player/65327e9c7ea42e125256b29a.md

```diff
@@ -0,0 +1,608 @@
+---
+id: 65327e9c7ea42e125256b29a
+title: Step 7
+challengeType: 0
+dashedName: step-7
+---
+
+# --description--
+
+All the songs are now in the `allSongs` array. The next thing you will need to do is to initialize the <dfn>Web Audio API</dfn>.
```

Comment:
We don't need to wrap Web Audio API in definintion tags since we are not defining it.

```suggestion
All the songs are now in the `allSongs` array. The next thing you will need to do is to initialize the `Web Audio API`.
```